woocommerce/woocommerce-ios
# Update color of buttons in navigation bar and other text buttons

Closes: #3251 

⚠️ Please note this PR is not against develop, but against the feature branch for UL&S 🙇


| Appearance | Before | After |
| ---- | ---- | ---- |
| Light | <img src="https://user-images.githubusercontent.com/2722505/100689360-47b8de00-33bf-11eb-9585-13a93dfdb56e.png" width="350"/> | <img src="https://user-images.githubusercontent.com/2722505/100689340-3ff93980-33bf-11eb-9f55-770893037ece.png" width="350"/> |
| Dark | <img src="https://user-images.githubusercontent.com/2722505/100689356-45ef1a80-33bf-11eb-891c-71ddecd7839f.png" width="350"/> | <img src="https://user-images.githubusercontent.com/2722505/100689335-3c65b280-33bf-11eb-9ff1-b9021afd0a6e.png" width="350"/> |

## Changes
* Update the values of `navButtonTextColor`, `textButtonColor` and `textButtonHighlightColor` passed to WPAuthenticator

I did not change the color of the title (i.e. "Log in"). Changing that takes one minute, @woocommerce/mobile-designers please let me know if that's something that should be updated.

## Testing
* Checkout the branch
* Run `bundle exec pod install`
* Log out if necessary
* Log in with site address. Notice the colours of the buttons in the navigation bar
Update release notes:
